draft-ietf-cdni-uri-signing.xml

```diff
@@ -608,6 +635,33 @@
           <section anchor="uri_container_forms_hash" title="URI Hash Container (uri-hash:)">
               <t>Prefixed with 'uri-hash:', this string is a URL Segment form (<xref target="RFC6920"/> Section 5) of the URI.</t>
           </section>
+
+          <section anchor="uri_container_forms_b" title="URI Base Container (b:)">
+              <t>This string is simply 'b:', with no additional
+              information. This form indicates that the URI MUST
+              match the Base claim of the JWT. A URI matches iff:
+              <list style="symbols">
+                  <t>If the scheme is present in the Base claim, it is,
+                  according to a case insensitive comparison, the same
+                  as the scheme of the URI; and</t>
+                  <t>If the host is present in the Base claim, it is,
```

Comment:
Seeing all these `if`'s I think the `iff` above should just be removed?

Reply:
Perhaps the ifs should turn into "not or"s? Would it be easier to read as: "Either the host is not present in the Base claim or it is, according to a case-insensitive comparison, the same as the host of the URI"?